language/install.py

```diff
@@ -290,6 +290,7 @@ def install_bindings(regent_dir, legion_dir, bindings_dir, python_bindings_dir,
             os.mkdir(build_dir)
         flags = (
             ['-DCMAKE_BUILD_TYPE=%s' % ('Debug' if debug else 'Release'),
+             '-DLegion_NETWORKS=%s' % os.environ.get('REALM_NETWORKS', ''),
```

Comment:
Could we do this in the style of the checks down the page from here? I.e.:

```
(['-DLegion_NETWORKS=%s' % os.environ['REALM_NETWORKS']] if 'REALM_NETWORKS' in os.environ else []) +
```

Feels appropriate to put this under `-DGASNet_CONDUIT`.

Reply:
Looks fine aside from a slight typo in the new code where a `)` has to be moved to the next line.